builds/jazz_azure-create-service/components/ApiApp.js

```diff
@@ -19,7 +19,7 @@ module.exports = class ApiApp {
                 await this.resourceFactory.createHostingPlan(this.data.resourceGroupName, 'westus', {}, 'Y1', this.data.appName);
                 await this.resourceFactory.createFunctionApp(this.data.appName, storageAccountKey, this.data.tags, undefined,undefined,undefined,undefined, this.data.runtime);
                 await this.resourceFactory.uploadZipToKudu(this.data.appName, this.data.zip);
-                await this.resourceFactory.createOrUpdateApiGatewayWithSwaggerJson(this.data.serviceName, this.data.apiId, this.data.swagger, this.data.basepath);
+                await this.resourceFactory.createOrUpdateApiGatewayWithSwaggerJson(this.data.serviceName, this.data.apiId, this.data.swagger, this.data.appName, this.data.basepath);
```

Comment:
Is appName a new attribute added to this method? What are the typical values? Is this only for Azure APIs?

Reply:
appName is one of the values provided in the payload. We are just passing that to this particular function in order to use it to update the swagger. Yes it is only for APIs